docs/DeveloperGuide.md

```diff
@@ -775,6 +805,41 @@ testers are expected to do more *exploratory* testing.
 
 ***
 
+### Adding a student
+
+1. Adding a student
```

Comment:
I think can remove this for all the test cases since it’s just repeating the header?